### PR TITLE
Add footer info/links

### PR DESCRIPTION
**Pull request by syzygy333**
It's a start.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -74,10 +75,12 @@ <h4 class="color-ratios--title">Most Readable Pair</h4>
...
+      <p><a title="Go to w3 website, link opens in a new window." target="_blank" href="http://www.w3.org/TR/WCAG20/">WCAG 2.0</a> level AA requires a contrast ratio of 4.5:1 for normal text and 3:1 for large text. Level AAA requires a contrast ratio of 7:1 for normal text and 4.5:1 for large text. Large text is defined as 14 point (typically 18.66px) and bold or larger, or 18 point (typically 24px) or larger.</p>
   
       <h3>Check your hover text color vs. your body text color:</h3>
-      <p>If <a title="Go to webaim.org, link opens in a new window." href="http://www.w3.org/TR/WCAG20/">links are not underlined</a>, there must be 3:1 contrast between link text and body text and also an additional change (e.g., underlining) on mouse hover and keyboard focus.</p>
+      <p>If <a title="Go to webaim.org, link opens in a new window." target="_blank" href="http://www.w3.org/TR/WCAG20/">links are not underlined</a>, there must be 3:1 contrast between link text and body text and also an additional change (e.g., underlining) on mouse hover and keyboard focus.</p>
```
Comment (jhogue) [inline index.html]: this does not go to web aim.org…

**Comment by jhogue** [comment]
Fine for now, a good start. 
/approved

**Comment by syzygy333** [comment]
Big changes to master incorporated. Re-fixes link to webaim, re-adds link to GH page

File: index.html [hunk 2]
```diff
@@ -3,7 +3,8 @@
   <head>
     <meta charset="UTF-8">
     <title>ColorCube: A color A11y Tool for Designers &amp; Developers</title>
-    <script src="https://use.fontawesome.com/4d174c70ae.js"></script>
+    <script src="https://use.fontawesome.com/e5751d5558.js"></script>
+    <link href="https://fonts.googleapis.com/css?family=Cabin:500,500i,700,700i" rel="stylesheet">
```
Comment (jhogue) [inline index.html]: Nope, remove Cabin. We don't need it anymore.

File: src/css/styles.css [hunk 3]
```diff
@@ -732,7 +738,11 @@ input[type="number"] {
   .button, .button:link, .button:visited {
     background-color: #f2f2f2;
     color: #666; }
-  .no-js .button:hover, .no-js .button:focus, .js.no-touchevents .button:hover, .js.no-touchevents .button:focus, .js.touchevents .button:active {
+  .no-js .button:hover,
+  .no-js .button:focus,
+  .js.no-touchevents .button:hover,
+  .js.no-touchevents .button:focus,
+  .js.touchevents .button:active {
```
Comment (jhogue) [inline src/css/styles.css]: How will this stuff work if we don't have these classes on the `<html>`?

File: index.html [hunk 4]
```diff
@@ -72,6 +73,7 @@ <h2 class="cc-branding__subtitle">Brand Palette Color Accessibility Checker from
         <p>ColorCube is an accessibility (a11y) tool aimed at designers and developers. From a list of colors, we give you lots of information about how those colors perform with common color combinations. </p>
         <p><strong>Why?</strong> A11y is a real issue that all users struggle with, and companies want to help. At Oomph we believe a better web is an accessible web for all. We wanted a tool that could be used before a project starts&mdash;one that helps assess how an existing brand color palette might perform in an accessible way. Starting early means that we can make adjustments early to avoid non-accessible colors later.</p>
         <p>This tool started as a prototype during a company-wide HackDay. We liked it so much, and saw so much potential, that we worked extra hard post-HackDay to improve it and release it to the world.</p>
+        <p>See it on <a title="Go to GitHub website, link opens in a new window." target="_blank"  href="https://github.com/oomphinc/colorcube"><span class="fa fa-github" aria-hidden="true"></span> GitHub</a>.</p>
```
Comment (jhogue) [inline index.html]: I'd rather have an <hr> above this and make it into like a copyright line…

2017 Oomph, Inc. Open source project with MIT license. [Contribute to the Github repo](link). 

Something like that.

File: src/css/styles.css (hunk 3 above)
Comment (syzygy333) [inline src/css/styles.css]: remnant from when I was adding a leaving-this-site arrow to all links with [target="_blank"]

Comment (syzygy333) [inline src/css/styles.css]: we do though

Comment (jhogue) [inline src/css/styles.css]: Duh, right, I was looking at <head> which was in view… <html> was not.

**Pull request merged**